### ISSUE TITLE
Script auto-update fails

### ISSUE THREAD

**Issue by Slivo-fr** [TS3Tools/TS3UpdateScript#114]
Hi there !

While trying to update I get the following error : 

```
Script Version: 5.8.4 (2021-05-21)
Script License:  (No support bought)
Support License:  (No support plan bought)

Donate and support this script: https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=7ZRXLSC2UBVWE

------------------------------------------------------------------

Please wait... Script is working...

Performing self-tests as preventive measures against possible failures                                                                                                                                                             [ OK ]

All collected information were validated successfully as preventive measures against possible failures                                                                                                                             [ INFO ]

Do you really want to update the script? ([y]es/[n]o) y
Script update is in progress...                                                                                                                                                                                       [ Should be updated ]
caution: filename not matched:  TS3UpdateScript-master/*
caution: excluded filename not matched:  TS3UpdateScript-master/configs/*
Cleaning up...t 'TS3UpdateScript-master/*': No such file or directory

Summary of measured execution time for each step
Total script: 14s
Self-Test: 4s
Execution Mechanism: 10s
Cleanup: 0s
```

**Comment by Sebi94nbg** [comment]
Hey,
Seems like I forgot to update some references to the `master` branch once I renamed it to `main`. My fault, sorry.

I've fixed this in https://github.com/TS3Tools/TS3UpdateScript/pull/116. I've also published a new release: [6.0.1](https://github.com/TS3Tools/TS3UpdateScript/releases/tag/6.0.1)

You can open the file `.updateScript.sh` and replace all `master` words with `main`. Afterwards, the update should work. :)

**Comment by Slivo-fr** [comment]
Thanks, trying out the 6.0.1 !
